### PR TITLE
Command/new members

### PR DESCRIPTION
**Pull request by Zynro**

### CODE REVIEW

File: modules/onmyoji/cogs/guildcmd.py [hunk 1]
```diff
@@ -105,42 +160,80 @@ def guild_json_writeout(self):
...
     async def schedule_set_message_error(self, ctx, error):
         if isinstance(error, commands.CheckFailure):
             await ctx.send("You do not have permission to uuse this command.")
 
+    @commands.command(name='newmem')
+    @commands.has_permissions(manage_roles=True)
+    async def new_member(self, ctx, member=None, nickname=None):
+        new_member_channel = self.bot.get_channel(config.new_member_channel_id)
+        #new_member_role = ctx.guild.get_role(config.new_member_role_id)
+        new_member_role = ctx.guild.get_role(177994521940983809)
```
Comment (wmak) [inline modules/onmyoji/cogs/guildcmd.py:210]: What's this magic number from?

File: modules/onmyoji/cogs/shard.py [hunk 2]
```diff
@@ -465,21 +479,21 @@ def check_trading_status(self, user):
...
+                self.shard_file_writeout_all();
                 return await ctx.send(f"{ctx.author.mention} is now available to be searched for trading.")
             trading_status = 'available' if  self.check_trading_status(ctx.author.id) else 'unavailable'
             return await ctx.send(f"{ctx.author.mention} is currently {trading_status} to be searched for trading.")
 
     @shard_status.command(name="on")
     async def shard_status_set_on(self, ctx):
         self.shard_trading_db[str(ctx.author.id)]['status'] = True
-        self.shard_file_writeout();
+        self.shard_file_writeout_all();
```
Comment (wmak) [inline modules/onmyoji/cogs/shard.py]: semicolons?

**Review by wmak — APPROVED** [review]
2 minor things, but LGTM

File: modules/onmyoji/cogs/guildcmd.py (hunk 1 above)
Comment (Zynro) [reply modules/onmyoji/cogs/guildcmd.py:210]: I just pulled it from Discord. I shouldn't have and i should dynamically do it by way of a command, but this is a temporary stopgap for now.

File: modules/onmyoji/cogs/shard.py (hunk 2 above)
Comment (Zynro) [reply modules/onmyoji/cogs/shard.py]: I have no idea why those are there, I guess I sneezed and Java came out.

**Pull request merged**